deeposlandia/static/app.js

```diff
@@ -0,0 +1,19 @@
+// Jitenshea
+
+// PREFIX should be defined in the settings.js file
+var API_URL = PREFIX + '/api';
+
+// Build the URL with a BASE_URL/<city> suffix based from a DOM element with the
+// "city" dataset attribute, i.e. 'data-city'.
+function cityurl(dom_id) {
```

Comment:
Legacy copy/pasting from jitenshea I think. You can remove it.

Reply:
You're write, I remove the file from the project. :)